Content.Server/Stack/StackSystem.cs

```diff
@@ -15,6 +15,7 @@ namespace Content.Server.Stack
     public sealed class StackSystem : SharedStackSystem
     {
         [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
+        [Dependency] private readonly IComponentFactory _compFactory = default!;
```

Comment:
```suggestion
        [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
```

we're gonna do this live because I have benchmarks running.